pachyderm-seldon/use-case/image-classification/experiment/data.py

```diff
@@ -38,7 +39,7 @@ def __getitem__(self, idx):
 # ======================================================================================================================
 
 
-def download_pach_repo(pachyderm_host, pachyderm_port, repo, branch, root, token):
+def download_pach_repo(pachyderm_host, pachyderm_port, repo, branch, root, token, project="default"):
```

Comment:
are you hardcoding project as "default"? Can we not pass that as part of the config (like what we do with repo)?